### PR TITLE
Work about section portfolio mobile

### PR DESCRIPTION
**Pull request by vikas924**
Hello there,

In this project, I have set up a "Portfolio-setup-and-mobilefirst" repository. My tasks include:

1. Documented a professional README.md.
2.Included a .gitignore file in main branch.
3.Created a new branch according to the GitHub flow rules.
4.Added files in new branch with seperate commits.
5.Added html file that includes works and about section for portfolio mobile first website.
6.Added css file that include styles for tworks and about section.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -128,3 +127,293 @@
...
+}
+
+.skills-head {
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  text-align: center;
+}
```
Comment (emmiiorji) [inline style.css:417]: Kindly adjust the skills section to match the design as closely as possible. Please, include the bullets beside the titles.

Your design:
image

Figma design:
image

File: style.css [hunk 2]
```diff
@@ -128,3 +127,293 @@
...
+.card {
+  display: flex;
+  flex-direction: column;
+  background-image: url(./Work-section-img/backgroundli.png);
+  height: 386px;
+  background-repeat: no-repeat;
+  background-size: 100%;
+  margin-bottom: 3.76%;
+  width: 100%;
+}
```
Comment (emmiiorji) [inline style.css]: Kindly adjust the project cards to centralize on the page as given on the design.

Your design:
image

Figma design:
image

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @vikas924, 
# Status: Changes Required ♻️

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

<div align='center'>

</div>

## Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Proper use of GitHub flow


## Required Changes ♻️
_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 2 above)
Comment (vikas924) [reply style.css]: Do i need to change inside content? or whole the container needs to be centralized.

Comment (emmiiorji) [reply style.css]: Hi @vikas924,

From the figma design, we should have equal width on both sides of the project card.

**Review by divinecharlotte — APPROVED** [review]
### Hi @vikas924 ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
### Status: Approved 👏  

image
### Good Highlights✔️

- Linter checks are passing
- Good commit messages
- Proper use of GitHub flow


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**